src/Core/Content/Sitemap/Provider/ProductUrlProvider.php

```diff
@@ -82,7 +82,7 @@ public function getUrls(SalesChannelContext $context, int $limit, ?int $offset =
             if (isset($seoUrls[$product['id']])) {
                 $newUrl->setLoc($seoUrls[$product['id']]['seo_path_info']);
             } else {
-                $newUrl->setLoc($this->router->generate('frontend.detail.page', ['productId' => $product['id']]));
+                $newUrl->setLoc(ltrim($this->router->generate('frontend.detail.page', ['productId' => $product['id']]), '/'));
```

Comment:
Why would you add a trailing slash here?

Reply:
This removes the trailing slash

Reply:
of course :see_no_evil:  sorry